entrypoint.sh

```diff
@@ -24,7 +24,8 @@ ${INPUT_AWS_REGION}
 text
 EOF
 
-BRANCH_NAME=$(echo $GITHUB_REF | cut -d'/' -f 3)
+REF_ARR=(${GITHUB_REF//// })
```

Comment:
replace / with space and create an array from the space-separated string

Reply:
can we do it like `echo $GITHUB_REF |  sed -e "s/\// /g"`

Reply:
Yes, done